src/components/tooltip/tooltip.tsx

```diff
@@ -57,8 +286,12 @@ export class Tooltip {
   @Prop({ reflect: true }) open = false;
 
   @Watch("open")
-  openHandler(): void {
-    this.reposition();
+  openHandler(value: boolean): void {
+    if (value) {
```

Comment:
We need to position/create popper before we open so that we can do the animation. I'll revert this part.